pysat/utils.py

```diff
@@ -780,6 +780,18 @@ def calc_solar_local_time(inst, lon_name=None, slt_name='slt'):
         data = inst.data.assign(pysat_slt=(inst.data.coords.keys(), slt))
         data.rename({"pysat_slt": slt_name}, inplace=True)
         inst.data = data
+
```

Comment:
Really for lines just above. Does inst[slt_name] = pds.Series(slt, index=inst.data.index) not work for both an underlying pandas or xarray format?

Reply:
I think it didn't, because of mixing Series with xarray.  I believe the way it is now, it doesn't care which one is used.